src/containers/org/components/AssignRoleModalBody.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import React, { useMemo } from 'react';
+
+import { List } from 'immutable';
+import { AuthUtils } from 'lattice-auth';
+import { Select, Typography } from 'lattice-ui-kit';
+import { PersonUtils } from 'lattice-utils';
+
+import { ModalBody } from '../../../components';
+import { getUserProfileLabel } from '../../../utils/PersonUtils';
+import type { ReactSelectOption } from '../../../types';
+
+const { getUserId } = PersonUtils;
+
+type Props = {
+  onChange :(value :any) => void;
+  members :List;
+}
+const AssignRoleModalBody = ({ onChange, members } :Props) => {
+
+  const thisUserInfo = AuthUtils.getUserInfo() || { id: '' };
+  const thisUserId = thisUserInfo.id;
+
+  const memberOptions = useMemo(() => {
+    const options = [];
+    members.forEach((member) => {
+      options.push({
+        label: getUserProfileLabel(member, thisUserId),
+        value: getUserId(member)
+      });
+    });
+    return options;
+  }, [members, thisUserId]);
+
```

Comment:
this should probably be members.hashCode() since `members` will be different on every render.

Reply:
it will not be. members is an immutable list of all members in the organization

Reply:
Yes that's true, but I think useMemo will treat it as a new object (I'm probably misremembering)